### PR TITLE
SubFleet and Aircraft Pages

### PR DESCRIPTION
**Pull request by FatihKoz**
Added subfleets , single subfleet and aircraft listing/displaying pages

Both blades do follow user settings for displaying values (like distance, fuel weight etc)

### CODE REVIEW

File: app/Providers/RouteServiceProvider.php [hunk 1]
```diff
@@ -128,6 +128,11 @@ private function mapWebRoutes()
                 Route::get('flights/search', 'FlightController@search')->name('flights.search');
                 Route::resource('flights', 'FlightController');
 
+                // Subfleets
+                Route::get('subfleets', 'SubFleetController@index')->name('subfleets.index');
```
Comment (nabeelio) [inline app/Providers/RouteServiceProvider.php:132]: Rename to `SubfleetController` so this matches how it's used elsewhere

File: resources/lang/en/common.php [hunk 2]
```diff
@@ -28,7 +29,14 @@
     'subfleet'       => 'Subfleet',
     'distance'       => 'Distance',
     'fuel'           => 'Fuel',
+    'fuelonboard'    => 'Fuel On Board',
```
Comment (nabeelio) [inline resources/lang/en/common.php:32]: Add these into the other lang files too with Google Translate or something

Comment (FatihKoz) [reply resources/lang/en/common.php:32]: This is why i hate translations :)

File: app/Providers/RouteServiceProvider.php (hunk 1 above)
Comment (FatihKoz) [reply app/Providers/RouteServiceProvider.php]: Done

File: resources/views/layouts/default/subfleets/aircraft_pireps.blade.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    <th>{{ trans_choice('common.pilot', 1) }}</th>
+  </tr>
+  @foreach($pireps as $pirep)
+    <tr>
+      <td class="text-left">{{ $pirep->airline->code }} {{ $pirep->ident }}</td>
+      <td><a href="{{ route('frontend.airports.show', [$pirep->dpt_airport_id]) }}">{{ $pirep->dpt_airport_id }}</a></td>
+      <td><a href="{{ route('frontend.airports.show', [$pirep->arr_airport_id]) }}">{{ $pirep->arr_airport_id }}</a></td>
+      <td>
+        @if($pirep->distance > 0)
+          @if(setting('units.distance') == 'km')
```
Comment (nabeelio) [inline resources/views/layouts/default/subfleets/aircraft_pireps.blade.php:19]: `===`

File: resources/lang/en/common.php (hunk 2 above)
Comment (nabeelio) [reply resources/lang/en/common.php:32]: Haha me too

Comment (FatihKoz) [reply resources/lang/en/common.php:32]: Done

File: resources/views/layouts/default/subfleets/aircraft_pireps.blade.php (hunk 3 above)
Comment (FatihKoz) [reply resources/views/layouts/default/subfleets/aircraft_pireps.blade.php]: Done

**Pull request closed without merging**